Running Issue

Build completes in 2 seconds, but when I try to run it, it fails with the following error .
atiya@atiya-Virtual-Machine:~/agreementmaker$ ./gradlew build

BUILD SUCCESSFUL in 2s
53 actionable tasks: 53 up-to-date
atiya@atiya-Virtual-Machine:~/agreementmaker$ ./gradlew run

> Task :ui-glue:run FAILED
log4j:WARN No appenders could be found for logger (am.app.Core).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
ERROR StatusLogger Log4j2 could not find a logging implementation. Please add log4j-core to the classpath. Using SimpleLogger to log to the console...
ERROR WordNetUtils java.lang.RuntimeException: Cannot find WordNet dictionary files. Expected the files to exist in this directory: /home/atiya/agreementmaker/AM_ROOT/wordnet-3.0
 java.lang.RuntimeException: Cannot find WordNet dictionary files. Expected the files to exist in this directory: /home/atiya/agreementmaker/AM_ROOT/wordnet-3.0
        at am.app.wordnet.WordNetUtils.initWordnet(WordNetUtils.java:48)
        at am.app.wordnet.WordNetUtils.<init>(WordNetUtils.java:24)
        at am.matcher.bsm.BaseSimilarityMatcher.<init>(BaseSimilarityMatcher.java:47)
        at am.app.mappingEngine.registry.MatcherRegistryImpl.<init>(MatcherRegistryImpl.java:60)
        at am.ui.glue.StartAgreementMaker.main(StartAgreementMaker.java:10)
Exception in thread "main" java.lang.RuntimeException: Cannot find WordNet dictionary files. Expected the files to exist in this directory: /home/atiya/agreementmaker/AM_ROOT/wordnet-3.0
        at am.app.wordnet.WordNetUtils.initWordnet(WordNetUtils.java:48)
        at am.app.wordnet.WordNetUtils.<init>(WordNetUtils.java:24)
        at am.matcher.bsm.BaseSimilarityMatcher.<init>(BaseSimilarityMatcher.java:47)
        at am.app.mappingEngine.registry.MatcherRegistryImpl.<init>(MatcherRegistryImpl.java:60)
        at am.ui.glue.StartAgreementMaker.main(StartAgreementMaker.java:10)

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':ui-glue:run'.
> Process 'command '/usr/lib/jvm/java-8-openjdk-amd64/bin/java'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s
35 actionable tasks: 1 executed, 34 up-to-date

--- a/README.md
+++ b/README.md
@@ -32,7 +32,7 @@ NOTE: On your computer the path `/path/to/agreementmaker/AM_ROOT/wordnet-3.0` wi
 
 You can ensure that the files are there by downloading them from the [WordNet website](https://wordnet.princeton.edu/download/current-version) (substitute `/path/to/agreementmaker/AM_ROOT/wordnet-3.0` with the real path in the error message):
 
-```
+```shell script
 mkdir -p /path/to/agreementmaker/AM_ROOT/wordnet-3.0
 cd /path/to/agreementmaker/AM_ROOT/wordnet-3.0
 wget http://wordnetcode.princeton.edu/wn3.1.dict.tar.gz

--- a/README.md
+++ b/README.md
@@ -20,6 +20,25 @@ To build Agreementmaker, use Gradle:
 
 ## Running
 
+### WordNet Dictionary Files
+
+AgreementMaker requires the WordNet dictionary files at application startup.  If it cannot find them it will show an error like this:
+
+```
+ERROR WordNetUtils java.lang.RuntimeException: Cannot find WordNet dictionary files. Expected the files to exist in this directory: /path/to/agreementmaker/AM_ROOT/wordnet-3.0
+```
+
+NOTE: On your computer the path `/path/to/agreementmaker/AM_ROOT/wordnet-3.0` will be a real path.
+
+You can ensure that the files are there by downloading them from the [WordNet website](https://wordnet.princeton.edu/download/current-version) (substitute `/path/to/agreementmaker/AM_ROOT/wordnet-3.0` with the real path in the error message):
+
+```
+mkdir -p /path/to/agreementmaker/AM_ROOT/wordnet-3.0
+cd /path/to/agreementmaker/AM_ROOT/wordnet-3.0
+wget http://wordnetcode.princeton.edu/wn3.1.dict.tar.gz
+tar --strip-components=1 -xvzf wn3.1.dict.tar.gz
+```
+
 To run AgreementMaker, use the `run` Gradle task:
 
 ```shell script

